docker/Dockerfile

```diff
@@ -20,7 +20,11 @@
 #
 # Start a container interactively, opening a bash in `/neo-python`, and mounting the current directory as `/neo-python/sc`:
 #
-#    $ docker run --rm -it -v $(pwd):/neo-python/sc --net=host -h neopython --name neopython neopython /bin/bash
+#    $ docker run --rm -it --entrypoint "/bin/bash" -v $(pwd):/neo-python/sc --net=host -h neopython --name neopython neopython
```

Comment:
I'm not sure about this -- making the default usage more complex. Perhaps it would be better to leave the default entrypoint as is, and use the `--entrypoint` arg when running the bootstrap version,